pkg/analysis/jsontags/testdata/src/a/a.go

```diff
@@ -18,6 +18,11 @@ type JSONTagTestStruct struct {
 	JSONTagWithID        string `json:"jsonTagWithID"`
 	JSONTagWithTTL       string `json:"jsonTagWithTTL"`
 	JSONTagWithGiB       string `json:"jsonTagWithGiB"`
+	Ignored              string `json:"-"`
+
+	IgnoredAnonymousStruct struct {
+		A string `json:""` // want "field A has empty json tag"
```

Comment:
Don't we want to ignore fields within an ignored struct?

Reply:
Make sense to me. Maybe, is it better to use inspect.WithStack ?

Reply:
I think it might be yeah, I'm wondering if we want to create a common helper to provider the inpect and filter out things like these fields we don't/can't reach 🤔 That might be a much larger refactor though

Reply:
> I think it might be yeah, I'm wondering if we want to create a common helper to provider the inpect and filter out things like these fields we don't/can't reach 🤔 That might be a much larger refactor though

It sounds good, but, as you said, it'll be much larger refactoring.
In here, I'll use WithStack for now.